saleor/webhook/validators.py

```diff
@@ -0,0 +1,46 @@
+import re
+from typing import Dict
+
+from django.core.exceptions import ValidationError
+
+HEADERS_NUMBER_LIMIT = 5
+HEADERS_LENGTH_LIMIT = 78
+
+
+def custom_headers_validator(headers) -> Dict[str, str]:
```

Comment:
Would be good to add a typing here, as someone could use the method with `None` (by mistake).